src/injected/visualization/center-position-calculator.ts

```diff
@@ -80,7 +80,7 @@ export class CenterPositionCalculator {
         );
 
         const myDocument = this.drawerUtils.getDocumentElement();
-        const body = myDocument.querySelector('body');
+        const body = myDocument.querySelector('html');
 
         const bodyStyle = this.windowUtils.getComputedStyle(body);
```

Comment:
so now... we'll get the computed style from the `<html>` element... does that just work the same as getting the computed styles from the `<body>` element